.github/workflows/openjdk_14_build.yml

```diff
@@ -0,0 +1,117 @@
+name: openjdk_14_build
+on:
+  pull_request:
+  push:
+    branches: [ main, develop ]
+    tags: [ v* ]
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    env:
+      MAVEN_CLI_OPTS: --batch-mode --no-transfer-progress
```

Comment:
I believe you still need to put `./mvnw $MAVEN_CLI_OPTS` for this to work. There's no built-in environment variable that maven uses for extra flags.

Reply:
doh, fixing